ui/src/app/base/components/MinimumKernelSelect/MinimumKernelSelect.tsx

```diff
@@ -37,7 +37,6 @@ export const MinimumKernelSelect = ({
       options={[
         {
           label: "Select minimum kernel",
-          value: "",
```

Comment:
drive by fix - we never actually want this option selectable, but the API refers to an empty string as "No minimum kernel"